slides.md

```diff
@@ -1,106 +1,134 @@
-class: center, title-page
+---
+theme: slidev-theme-nearform
+layout: image-right
+highlighter: shiki
+lineNumbers: false
+---
+
+<img class=logo src="/images/nearform.svg">
 
 # The Fastify Workshop
 
-<img src="assets/fastify.png" style="width: 30%">
+<img src="assets/fastify.png" style="width: 30%;">
 
----
+<div class="copyright">
+
+© Copyright 2019-2022 NearForm Ltd. All Rights Reserved.
 
-class: branded
+</div>
+
+---
 
 # Introduction: Why Fastify
 
+<div class="flex-row">
+
 - An efficient server implies a lower cost of the infrastructure, a better responsiveness under load and happy users
 
 - How can you efficiently handle the resources of your server, knowing that you are serving the highest number of requests possible, without sacrificing security validations and handy development?
 
----
+</div>
 
-class: branded
+---
 
 # Introduction: Why Fastify /2
 
-- Fastify is a Node.js web framework focused on performance and developer experience
+<div class="flex-row">
 
+- Fastify is a Node.js web framework focused on performance and developer experience
 - The Fastify team has spent considerable time building a highly supportive and encouraging community
 
 - Fastify gained adoption by solving real needs of Node.js developers
 
----
+</div>
 
-class: branded
+---
 
 # Core features
 
+<div class="flex-row">
+
 - **Highly performant**: as far as we know, Fastify is one of the fastest web frameworks in town, depending on the code complexity we can serve up to 30 thousand requests per second
 
 - **Extensible**: Fastify is fully extensible via its hooks, plugins and decorators
 
 - **Schema based**: even if it is not mandatory we recommend to use JSON Schema to validate your routes and serialize your outputs, internally Fastify compiles the schema in a highly performant function
 
----
+</div>
 
-class: branded
+---
 
 # Core features /2
 
+<div class="flex-row">
+
 - **Logging**: logs are extremely important but are costly; we chose the best logger to almost remove this cost, Pino!
 
 - **Developer friendly**: the framework is built to be very expressive and to help developers in their daily use, without sacrificing performance and security
 
 - **TypeScript ready**: we work hard to maintain a TypeScript type declaration file so we can support the growing TypeScript community
 
----
+</div>
 
-class: branded
+---
 
 # Who is using Fastify
 
+<div class="flex-row">
+
 ![Who is using Fastify](assets/who.png)
 
 https://www.fastify.io/organisations/
 
----
+</div>
 
-class: branded
+---
 
 # Ecosystem
 
+<div class="flex-row">
```

Comment:
All the slides need this class? Wouldn't be better to adjust it automatically in the theme?

Reply:
It turns out that even if I don't use the class the slides look the same. Seems like this is the default class 🙂